vendors/rakwireless/codecs/test_decode_example.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
This file must be named: `test_decode_rak_standard.json` (as it is related to the `rak_standard.js` codec file).

Reply:
OK, got it.